app/src/main/java/com/codesoom/assignment/TaskAdvice.java

```diff
@@ -0,0 +1,60 @@
+package com.codesoom.assignment;
+
+import com.codesoom.assignment.exceptions.TaskNotFoundException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+import org.springframework.web.context.request.WebRequest;
+
+import java.util.Date;
+
+/**
+ * 할 일들의 예외처리를 위한 클래스
```

Comment:
클래스에 대한 설명에 `클래스`라 적을 필요는 없습니다.
```suggestion
 * 할 일들의 예외처리를 담당합니다.
```